### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by DonBraulio**
This PR promotes the `toy_example.ipynb` -> `getting_started.ipynb` after some improvements to it:
- Added general explanations.
- Added links to resources in the documentation (Next steps section)
- Added graph definition explanations and more examples.
- Cover basic index manipulation example.

I'm also proposing to link it from the `README`, with a couple more changes there. Further discussion about those is expected and welcome 🙂 

Note: This should be merged after #141 , since it introduces some changes to the toy_example notebook.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,19 +34,26 @@ sma = tp.simple_moving_average(node, window_length=tp.days(7))
 result = sma.evaluate(evset)
```
Comment (achoum) [inline README.md:34]: What about adding a print and plot?

>> tp.read_event_set("path/to/temporal_data.csv", timestamp_column="time")
What about showing / describing what the csv file contains. Alternatively, what about creating an example by hand.

File: README.md [hunk 2]
```diff
@@ -36,19 +34,26 @@ sma = tp.simple_moving_average(node, window_length=tp.days(7))
...
 ```
 
+Check the [Getting Started tutorial](examples/tutorials/getting_started.ipynb) to try it!
```
Comment (achoum) [inline README.md:37]: This link does not exist anymore, right?

Comment (achoum) [inline README.md:37]: Should this point to the website or github?

**Review by achoum — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (DonBraulio) [reply README.md:37]: Sorry, this one should point to the docs website, definitely

Comment (ianspektor) [inline README.md:37]: nit: to try it out / to give it a try?

File: README.md [hunk 3]
```diff
@@ -36,19 +34,26 @@ sma = tp.simple_moving_average(node, window_length=tp.days(7))
...
+
 ## Key features
 
 These are what set Temporian apart.
 
 - **Simple and powerful API**: Temporian exports high level operations making processing complex programs short and ready to read.
+- **Flexible data model**: Temporian models temporal data as a sequence of events, supporting non-uniform sampling timestamps seamlessly.
```
Comment (ianspektor) [inline README.md:44]: nice

File: README.md [hunk 4]
```diff
@@ -36,19 +34,26 @@ sma = tp.simple_moving_average(node, window_length=tp.days(7))
...
 - **Prevents modeling errors**: Temporian programs are guaranteed not to have future leakage unless the user calls the `leak` function, ensuring that models are not trained on future data.
 - **Iterative development**: Temporian can be used to develop preprocessing pipelines in Colab or local notebooks, allowing users to visualize results each step of the way to identify and correct errors early on.
 - **Efficient and well-tested implementations**: Temporian contains efficient and well-tested implementations of a variety of temporal data processing functions. For instance, our implementation of window operators is **x2000** faster than the same function implemented with NumPy.
 - **Wide range of preprocessing functions**: Temporian contains a wide range of preprocessing functions, including moving window operations, lagging, calendar features, arithmetic operations, index manipulation and propagation, resampling, and more. For a full list of the available operators, see the [operators documentation](https://temporian.readthedocs.io/en/latest/reference/temporian/core/operators/).
 
-## Documentation
+## Why Temporian?
```
Comment (ianspektor) [inline README.md:50]: I'd kill this section, especially since the M5 notebook isn't really readable atm

File: README.md [hunk 5]
```diff
@@ -4,13 +4,9 @@
 
 **Temporian** is a Python package for **feature engineering of temporal data**, focusing on providing a **simple and powerful API**, a first-class **iterative development** experience, **efficient and well-tested implementations** of common and not-so-common temporal data preprocessing functions, and **preventing common modeling errors**.
 
-## Why Temporian?
-
-Temporian helps you **focus on high-level modeling**.
-
-Temporal data processing is commonly done with generic data processing tools. However, this approach is often tedious, error-prone, and requires engineers to learn and re-implement existing methods. Additionally, the complexity of these tools can lead engineers to create less effective pipelines in order to reduce complexity. This can increase the cost of developing and maintaining performant ML pipelines.
+## Documentation
```
Comment (ianspektor) [inline README.md:7]: would definitely move after the installation part, and I think after the getting started too

**Review by ianspektor — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (DonBraulio) [reply README.md:59]: Agree. Removed 😬

**Comment by DonBraulio** [comment]
> Should this folder be ignored?

It's already ignored via `tmp_*` in gitignore.

**Comment by DonBraulio** [comment]
README fixed and comments addressed. Make sure to force refresh on the docs preview to see the changes.

**Pull request merged**